src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -353,7 +353,10 @@ public ComputeEngineInstance provision() throws IOException {
     /** Initializes transient properties */
     protected Object readResolve() {
         labelSet = Label.parse(labels);
-        this.networkInterfaceIpStackMode = new NetworkInterfaceSingleStack(externalAddress);
+        if (externalAddress != null) {
+            this.networkInterfaceIpStackMode = new NetworkInterfaceSingleStack(externalAddress);
+            this.externalAddress = null;
+        }
```

Comment:
The `readResolve` method is used for retrocompatibility but it was broken because we pass in it each time we `Save` the instance.
With this modification, we'll set the `networkInterfaceIpStackMode` only one time.